dbms/src/Interpreters/Settings.h

```diff
@@ -240,6 +241,8 @@ struct Settings
     \
     M(SettingInt64, remote_checkpoint_interval_seconds, 30, "The interval of uploading checkpoint to the remote store. Unit is second.")                                                                                                \
     M(SettingInt64, remote_gc_interval_seconds, 3600, "The interval of running GC task on the remote store. Unit is second.")                                                                                                           \
+    M(SettingDouble, remote_gc_ratio, 0.2, "The files with valid rate less than this threshold will be compacted")                                                                                                                      \
```

Comment:
Hmmm it is too small IMO. I think 0.5 is more suitable. If it is proven to be not cost-efficient, we can tune it later.

Reply:
change to 0.5 by default